### ISSUE TITLE
The specification uses text quotes and markdown quotes interchangably

### ISSUE THREAD

**Issue by constantinpape** [ome/ngff#38]
Currently, in the spec text quotes "some_name" and markdown quoutes `some_name` are used interchangeably; for example sometimes "mutliscales" is used, sometimes `multiscales`. It would be nice to be more stringent about this and add a recommendation which quotes to use.

**Comment by joshmoore** [comment]
Agreed. Sounds like a "style-guide" section. There is likely something within bikeshed itself, e.g. https://tabatkins.github.io/bikeshed/#dfn-types but I don't know if if it's use will be too verbose.

Failing finding a proper markup, I think I'd err on the side of markdown quotes to define class/object-style information.

**Comment by joshmoore** [comment]
> I think I'd err on the side of markdown quotes to define class/object-style information

Ah, I think I see what I was (at least sometimes) trying to get across: in some cases it's referring to the json-key instead of the concept. Perhaps that's too confusing to get across and we should just consistently refer to _either_ the type _or_ the json item.

**Comment by jo-mueller** [comment]
@lubianat I started writing a [contribution guide](https://ngff-spec.readthedocs.io/en/latest/contribute/) with some styling hints over at the ngff-spec repo. With https://github.com/ome/ngff/pull/410 merged, maybe we could move this bit upstream into the ngff repo as well. Feedback from the doc team about this part:

While hints on formatting things are important and welcome, we should make sure that the contribution pages are encouraging in the sense that people feel confident about being able to contribute.
